ppml/trusted-machine-learning/spark-executor-template.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
```

Comment:
If we have duplicate yaml, we need to merge them.

Reply:
Yamls for driver and executor are different, for example the driver requires a kubeconfig file to create pod on k8s while the executor does not.